pleskdistup/actions/extensions.py

```diff
@@ -164,3 +164,74 @@ def estimate_prepare_time(self) -> int:
 
     def estimate_revert_time(self) -> int:
         return 20
+
+
+class AssertPleskExtensions(action.CheckAction):
+    installed: typing.Set[str]
+    not_installed: typing.Set[str]
+    installed_description: str
+    not_installed_description: str
+    installed_violations: typing.Set[str]
+    not_installed_violations: typing.Set[str]
+    _name: str
+
+    def __init__(
+        self,
+        installed: typing.Optional[typing.Iterable[str]] = None,
+        not_installed: typing.Optional[typing.Iterable[str]] = None,
+        name: str = "check Plesk extensions state:",
+        installed_description: typing.Optional[str] = None,
+        not_installed_description: typing.Optional[str] = None,
+    ):
+        if installed is None and not_installed is None:
+            raise ValueError("'installed' and 'not_installed' can't be both None")
+        self.installed = set(installed) if installed is not None else set()
+        self.not_installed = set(not_installed) if not_installed is not None else set()
+        self._name = name
+        self.installed_violations = set()
+        self.not_installed_violations = set()
+        if installed_description is None:
+            self._installed_description = "Required Plesk extensions are missing. To continue, please install the following extensions:\n\t- {installed_violations}"
```

Comment:
If `installed_description`, `not_installed_description` aren't `None`, they are not going anywhere. The same bug is in `AssertPleskComponents`.